tb_plugin/torch_tb_profiler/profiler/node.py

```diff
@@ -166,6 +171,10 @@ def get_node_argument(event):
         if event.type == EventTypes.KERNEL:
             kwargs["blocks_per_sm"] = event.args.get("blocks per SM", 0)
             kwargs["occupancy"] = event.args.get("est. achieved occupancy %", 0)
+            kwargs["grid"] = str(event.args.get("grid", "None"))
```

Comment:
keep the original value by removing the str

Reply:
Thanks, done.